bindings/jupyroot/python/JupyROOT/helpers/utils.py

```diff
@@ -40,28 +40,63 @@
 _jsNotDrawableClassesPatterns = ["TEve*","TF3","TPolyLine3D"]
 
 
-_jsROOTSourceDir = "/static/"
 _jsCanvasWidth = 800
 _jsCanvasHeight = 600
 
 _jsCode = """
 <div id="{jsDivId}"
      style="width: {jsCanvasWidth}px; height: {jsCanvasHeight}px">
 </div>
-<script src="/static/components/requirejs/require.js" type="text/javascript" charset="utf-8"></script>
```

Comment:
Again, I do not understand why need to load require.js - which probably not there at all.
Just instead this line replace:
```
<script src="/static/scritps/JSRootCore.js" type="text/javascript"></script>
```
And display function can be very simple:
```
function display_{jsDivId}() {{
   var obj = JSROOT.JSONR_unref({jsonContent});
   JSROOT.key_handling = false;
   JSROOT.draw("{jsDivId}", obj, "{jsDrawOptions}");
}}
```

Reply:
Or if logic with baseUrl should be preserved - load `/static/scripts/JSRootCore.js` instead of require.js.

Reply:
The line that loads requirejs was added by @philipleindecker (summer student) to make the JSRoot graphics work in JupyterLab, which does not load requirejs, unlike the classic Jupyter.

After that, what @diocas did (this PR) is differentiate between the JupyterLab case and the classic Jupyter case.

If you think the requirejs loading is not necessary for JSRoot in JupyterLab, it would be nice to have a simplification of the code as you propose. But please test it first both with JupyterLab and classic Jupyter.